acquia/acquia_cms
# ACMS-1066: Upgrade contrib modules

**Motivation**
Fixes #[ACMS-1066](https://backlog.acquia.com/browse/ACMS-1066)

**Proposed changes**
Upgrade contrib Modules:
1. Address: 8.x-1.10 
2. Admin Toolbar: 3.1.0
3. File metadata manager: 8.x-2.3
4. Geocoder: 8.x-3.25
5. Pathauto: 8.x-1.9
6. Search API: 8.x-1.23
7. Search API Solr: 4.2.7
8. Simple XML sitemap: 4.1.1
9. Shield: 8.x-1.6

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch.
- Visit module page and verify search api and search api solr module are update.

**Merge requirements**
